Sources/AppcuesKit/Presentation/Debugger/FloatingView/DebugView.swift

```diff
@@ -24,8 +24,8 @@ internal class DebugView: UIView {
 
     private var initialTouchOffsetFromCenter: CGPoint = .zero
 
-    private var dismissViewTimer: Timer?
-    private var canDismiss: Bool { dismissViewTimer?.isValid == false }
+    private var dismissViewTimer: Task<Void, Never>?
```

Comment:
should the variable be renamed?

Reply:
It's still functioning as a timer, so I don't mind the name since it signals the intended purpose 🤷‍♂️

Reply:
sounds good 👍